tools/cli/factory.go

```diff
@@ -58,7 +58,7 @@ type clientFactory struct {
 
 // NewClientFactory creates a new ClientFactory
 func NewClientFactory() ClientFactory {
-	logger := log.NewDefaultLogger()
+	logger := log.NewTestLogger()
```

Comment:
There are few cases like this where default logger is used from CLI. I would copy `NewTestLogger` to `NewCLILogger` and use it everywhere in CLI. In future these two might be different.

Reply:
Added NewCLILogger()